src/workspace.cpp

```diff
@@ -14,6 +15,13 @@ QList<Workspace> Workspace::filterByCurrentScreen(QList<Workspace> inputWorkspac
     return outputWorkspaces;
 }
 
+QList<Workspace> Workspace::orderByName(QList<Workspace> inputWorkspaces) {
+    std::sort(inputWorkspaces.begin(), inputWorkspaces.end(), [](Workspace a,Workspace b){
+            return (a.name < b.name);  //NOTICE as the variables are named right now the full name is copied on the id!
+            });
```

Comment:
```suggestion
    std::sort(inputWorkspaces.begin(), inputWorkspaces.end(), [](Workspace a,Workspace b) {
        return (a.name < b.name);  //NOTICE as the variables are named right now the full name is copied on the id!
    });
```

One more formatting nit :sweat_smile:

Reply:
Yeah, done (also, I removed the silly comment on the same line).
May I propose the use of a formatter like clang-format to avoid such inconveniences in the future?
You can have a `.clang-format` file and dictate a specific style for the whole project.E.g.  Look at `LambdaBodyIndentation` from [clang-format](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)

Reply:
Thanks! We already have that though we don't specify the `LambdaBodyIndentation`, sorry about that. If you wanted to make a PR to specify that, it would be awesome! You also seem to have left two spaces after the end of the line, which isn't ideal, we should set up a rule for that as well...